### DISCUSSION TITLE
Firefox store removed

### DISCUSSION THREAD

**Discussion by xtexx** [jiacai2050/gooreplacer4chrome#139] [Q&A]
现在在Firefox已经找不到扩展了

**Comment by jiacai2050** [comment]
没想到插件被火狐移除了，影响到这么多人了。

已经再和他们沟通了，恢复时间未知

**Comment by jiacai2050** [comment]
<img width="942" alt="image" src="https://user-images.githubusercontent.com/3848910/205934415-ec7164d2-8bb7-49f1-910b-67a253508097.png">
无语了，沟通了几次都是失败，大家可以下载 release 里面的 zip 包，解压后，加载使用吧，不想和 mozilla 的人折腾了。主要分两步：

1. 先选择 `Debug Add-ons`
2. 再选择 `Load temporary Add-on..`
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/3848910/205934850-fffb7908-5868-496e-8d54-65734be20dcd.png">
<img width="762" alt="image" src="https://user-images.githubusercontent.com/3848910/205934949-877fe1ef-e008-401a-957e-0c4462778d26.png">

# 最新 release
- https://github.com/jiacai2050/gooreplacer/releases/tag/v3.13.0

**Comment by ShadowEmbrace** [comment]
现在火狐的插件是别人搬运的么？
https://addons.mozilla.org/en-US/firefox/addon/joeli/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search

**Comment by jiacai2050** [comment]
应该是的

**Comment by jiacai2050** [comment]
![SCR-20230330-tepz](https://user-images.githubusercontent.com/3848910/228865742-373cf850-4783-49a3-afc1-290245ce99d2.png)


diff-so-fancy 结果

**Reply by jiacai2050** [reply]
<img width="1668" alt="image" src="https://user-images.githubusercontent.com/3848910/230119253-2bedd850-b252-4f42-b454-ba872ae9b65e.png">

**Comment by jiacai2050** [comment]
https://github.com/jiacai2050/my-works/blob/main/gooreplacer
已经用 JS 重写，重新上传到 Firefox 那边了。
